scipy/optimize/_linprog_util.py

```diff
@@ -1424,28 +1432,18 @@ def _check_result(x, fun, status, slack, con, bounds, tol, message):
         message = ("The solution does not satisfy the constraints within the "
                    "required tolerance of " + "{:.2E}".format(tol) + ", yet "
                    "no errors were raised and there is no certificate of "
-                   "infeasibility or unboundedness. This is known to occur "
-                   "if the `presolve` option is False and the problem is "
-                   "infeasible. This can also occur due to the limited "
-                   "accuracy of the `interior-point` method. Check whether "
+                   "infeasibility or unboundedness. Check whether "
                    "the slack and constraint residuals are acceptable; "
-                   "if not, consider enabling presolve, reducing option "
-                   "`tol`, and/or using method `revised simplex`. "
-                   "If you encounter this message under different "
-                   "circumstances, please submit a bug report.")
-    elif status == 0 and contains_nans:
```

Comment:
Was it possible for this condition to be reached? Above, if `contains_nans` is `True`, then `is_feasible` is `False`, so code would enter the branch above, right?